js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/StatusDot.tsx

```diff
@@ -5,7 +5,8 @@ import {GraphNode} from '../Utils';
 
 export function StatusDot({node}: {node: Pick<GraphNode, 'assetKey' | 'definition'>}) {
   const {liveData} = useAssetLiveData(node.assetKey);
-  if (!liveData) {
+
+  if (!liveData || !node.definition) {
```

Comment:
If the node has no definition it could be a deleted asset in which case showing the loading dot is incorrect. Should we return null for that case? Or maybe have a new status dot that indicates the asset no longer exists?

Reply:
I suppose it could also be an external asset?

Reply:
Going to revert this particular change because I can't remember why I put it here